packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -62,6 +62,16 @@ export class Wallet {
     return await this.client.call('getbalance', ['*', minimumConfirmation, includeWatchOnly], 'bignumber')
   }
 
+  /**
+   * Returns an object with all balances
+   *
+   * @param {boolean} [withTokens=false] - Include tokens balances; Default is 'false' for backward compatibility.
+   * @return {Promise<WalletBalances>}
+   */
+  async getBalances (withTokens = false): Promise<WalletBalances> {
+    return await this.client.call('getbalances', [withTokens], 'bignumber')
+  }
```

Comment:
```suggestion
  async getBalances (): Promise<WalletBalances> {
    return await this.client.call('getbalances', [false], 'bignumber')
  }
```

You can go further and just remove that option entirely

Reply:
Removed.